### PR TITLE
working version

### PR DESCRIPTION
**Pull request by BvB-tech**

### CODE REVIEW

File: case1.php [hunk 1]
```diff
@@ -11,60 +11,149 @@
...
+			$energyUsage = $value;
+			if($energyUsage <= 50){
+				$energyCost = $energyUsage * 3.5;
+			}elseif($energyUsage > 50 && $energyUsage <= 150){
+				$energyCost = 175 + ($energyUsage - 50)* 4;
+
+			}elseif($energyUsage > 150 && $energyUsage <= 250){
+				$energyCost = 575 + ($energyUsage -150) * 5.2;
+			}elseif($energyUsage > 250){
+				$energyCost = 575 + ($energyUsage -250) * 6.5;
```
Comment (e10ne) [inline case1.php]: 575 veranderen naar 1095

File: case1.php [hunk 2]
```diff
@@ -11,60 +11,149 @@
...
+			
+			
+			$month = "";
+			$energyCost = "";
+			$energyTax = 0;
+			if (isset($_POST['month'])) { $month = getMonth($_POST['month']); }
+			if (isset($_POST['period'])) { $energyCost = energyUsageCalc($_POST['period']);}
+			if (isset($_POST['period'])) { $energyTax = tax($_POST['period']); }
+
+			echo " Uw verbruik is " . "in" . " "  . $month . "is" ." " . $energyCost . "<br>";
```
Comment (t-molder) [inline case1.php]: echo " Uw verbruik " . "in" . " "  . $month . "is  €" ." " . $energyCost . "<br>";

File: case1.php [hunk 3]
```diff
@@ -11,60 +11,149 @@
...
+			echo " de betaalde belasting is" . " " . $energyTax;
```
Comment (t-molder) [inline case1.php]: echo " de betaalde belasting is €" . " " . $energyTax;

File: case1-loop-experiment.php [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+			
+			$month = "";
+			$energyCost = "";
+			$energyTax = 0;
+			
+			// if (isset($_POST['month'])) { $month = getMonth($_POST['month']); }
+			if (isset($_POST['period'])) { $energyCost = energyUsageCalc($_POST['period']);}
+			if (isset($_POST['period'])) { $energyTax = tax($_POST['period']); }
+
+			echo " Uw verbruik is " . "in" . " "  . $month . "is" ." " . $energyCost . "<br>";
```
Comment (t-molder) [inline case1-loop-experiment.php:65]: echo " Uw verbruik in" . " "  . $month . "is €" ." " . $energyCost . "<br>";

File: case1-loop-experiment.php [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+			echo " de betaalde belasting is" . " " . $energyTax;
```
Comment (t-molder) [inline case1-loop-experiment.php:66]: echo " de betaalde belasting is €" . " " . $energyTax;

**Review by t-molder — DISMISSED** [review]

**Review by e10ne — APPROVED** [review]

**Pull request merged**